src/XR/features/WebXRPlaneDetector.ts

```diff
@@ -187,23 +189,29 @@ export class WebXRPlaneDetector extends WebXRAbstractFeature {
         }
     }
 
-    private _init() {
+    private _initPlaneDetector(session: XRSession) {
         const internalInit = () => {
             this._enabled = true;
             if (this._detectedPlanes.length) {
                 this._detectedPlanes.length = 0;
             }
         };
-        if (!this._xrSessionManager.session.updateWorldTrackingState) {
+
+        if (!!this._options.preferredDetectorOptions &&
+            !!session.trySetPreferredPlaneDetectorOptions) {

```

Comment:
I wonder - this function is added to the XRSession in babylon native, but is not included in WebXR.d.ts, which is the one most web users will use. Won't that fail with unknown function for web devs?